.github/workflows/deploy_app.yml

```diff
@@ -11,7 +11,7 @@ on:
         required: true
 
 env:
- DOCKERHUB_REPOSITORY: dfedigital/teaching-vacancies
+ DOCKERHUB_REPOSITORY: ghcr.io/dfe-digital/teaching-vacancies
```

Comment:
Should we rename this to `DOCKER_REPOSITORY`, or does something other than this workflow depend on it being called `DOCKERHUB_REPOSITORY`?